### PR TITLE
update(geoip-lite): correct v1.4

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- remove argument from `stopWatchingDataUpdate`
- simplify module expors and linter config
- correct author (owner) GitHub handle
- add maintainer
- amend tests

https://github.com/geoip-lite/node-geoip/blob/4f68410bbc47c7075c3c9ee7c84fbd42534e8e65/lib/geoip.js#L503

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-tessts) to reflect the change.
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes

### CODE REVIEW

File: types/geoip-lite/geoip-lite-tests.ts [hunk 1]
```diff
@@ -15,6 +14,16 @@ if (stringLookup) {
 // start the data update watcher
 geoip.startWatchingDataUpdate();
 
+//
```
Comment (witem) [inline types/geoip-lite/geoip-lite-tests.ts:17]: Need to delete this

**Review by witem — CHANGES_REQUESTED** [review]
Good work, thanks. Need one minor change

File: types/geoip-lite/geoip-lite-tests.ts (hunk 1 above)
Comment (peterblazejewicz) [inline types/geoip-lite/geoip-lite-tests.ts:17]:
```suggestion
```

Comment (peterblazejewicz) [reply types/geoip-lite/geoip-lite-tests.ts]: amended, thx!

**Review by witem — APPROVED** [review]
lgtm

**Comment by peterblazejewicz** [comment]
Ready to merge 💋

**Pull request merged**